dev/io.openliberty.jakarta.xmlBinding.3.0/.project

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <projectDescription>
-	<name>io.openliberty.jakarta.jaxws.3.0</name>
+	<name>io.openliberty.jakarta.xmlBinding.3.0</name>
```

Comment:
I assume this is just git being funny and mismatching the project names after the rename, but just want to point it out just in case.

Reply:
Yeah it is.  You can see it in the other one as well.  It did its best.